Library/Homebrew/cask/quarantine.rb

```diff
@@ -55,26 +55,58 @@ def status(file)
                      print_stderr: false).stdout.rstrip
     end
 
-    def cask(cask: nil, download_path: nil)
+    def disable_translocation(xattr)
+      fields = xattr.split(";")
+
+      # Fields: status, epoch, download agent, event ID
+      # Let's toggle the app translocation bit, bit 8
+      # http://openradar.me/radar?id=5022734169931776
+
+      fields[0] = (fields[0].to_i(16) | 0x0100).to_s(16).rjust(4, "0")
+
+      fields * ";"
```

Comment:
Although I love the `*` notation, can you use `fields.join(";")` as that’s much more commonly understood?